app/src/main/java/com/example/scoutconcordia/Activities/MapsActivity.java

```diff
@@ -171,18 +171,12 @@
     @Override protected void onCreate(Bundle savedInstanceState) {
 
 
-//        // Victor's log tests
-//        ShuttleInfo testShuttleInfo = new ShuttleInfo();
-//
-//        //This gets the estimated time in minutes from Loyola to SGW at the current time
-//        Log.w("ShuttleInfo", "ESTIMATED TIME IS:  " + testShuttleInfo.getEstimatedRouteTimeFromLoyola() + " minutes");
-//
-//        //This gets the estimated time in minutes from SGW to Loyola at the current time
-//        Log.w("ShuttleInfo", "ESTIMATED TIME IS:  " + testShuttleInfo.getEstimatedRouteTimeFromSGW() + " minutes");
+
 
 
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_maps);
+        setmContext(this);
```

Comment:
Can you explain what this does again and how does it solve the previous bug?

Reply:
This is a custom setter method that we have created in the MapsActivity class, the point being that we need to pass down the Context of a calling class towards a non-activity class, in order to have access to resource files. The ShuttleInfo class on its own cannot access the resource files such as shuttle times, hence it is necessary that it gets the context from the activity class which calls it.